crypto/threads_pthread.c

```diff
@@ -126,7 +126,16 @@ int CRYPTO_THREAD_init_local(CRYPTO_THREAD_LOCAL *key, void (*cleanup)(void *))
 
 void *CRYPTO_THREAD_get_local(CRYPTO_THREAD_LOCAL *key)
 {
+#ifndef CHARSET_EBCDIC  
     return pthread_getspecific(*key);
+#else    
+    /*  On z/OS this is 
+    int pthread_getspecific(pthread_key_t key, void **value); 
+    */
+    void *some_value;
+    pthread_getspecific(*key, &some_value);
+    return some_value;
```

Comment:
This doesn't seems to have anything to do with EBCDIC character set and has everything to do with z/OS - so this should probably be in a different PR and not use "CHARSET_EBCDIC" to decide what to do (not all EBCDIC systems are z/OS)

Reply:
Removed